DEVELOPMENT.md

```diff
@@ -76,12 +76,17 @@ export KO_DOCKER_REPO='gcr.io/my-gcloud-project-id'
 
 Make sure to configure
 [authentication](https://cloud.google.com/container-registry/docs/advanced-authentication#standalone_docker_credential_helper)
-for your `KO_DOCKER_REPO` if required. To be able to push images to
-`gcr.io/<project>`, you need to run this once:
+for your `KO_DOCKER_REPO` if required. You need to run this once if
```

Comment:
Can we hoist this up to the `Create a cluster and repo` section? It seems redundant. I noticed the link to docker hub's quick start is wrong. It links to repos, but it should be https://docs.docker.com/docker-hub/

If we fix that, the links in that section recommend doing either `gcloud auth configure-docker` or `docker login`. If we really want to inline the login instructions here, we could put them as sub-bullets under the quickstart links.

Reply:
Thanks. Add a **Note** under the quickstart links. Fixed the links to Docker Hub quick start.

Reply:
Thanks!